[realm_destroyed.py]
  [test_shared_worker]
    expected: FAIL
